docs/processors/builtin.mdx

```diff
@@ -0,0 +1,112 @@
+---
+title: 'Built-in processors'
+sidebar_position: 1
+---
+
+:::note
+Note that all built-in processors that operate on the payload actually operate on `Record.Payload.After`.
+If you need to manipulate the field `Record.Payload.Before` you can use
+a [JavaScript processor](./javascript-processors).
+:::
+
+Below is a list of available built-in processors in Conduit. An example usage of a built-in processor is available in
+[`extract-field-transform.sh`](https:github.com/ConduitIO/conduit/blob/main/examples/processors/extract-field-transform.sh).
+The script will run Conduit with a pipeline containing the built-in `extractfieldpayload` processor.
+
+### extractfieldkey
+
+- If the key is raw and has a schema attached, extracts the field and uses it to replace the entire key.
+- If the key is raw and has no schema, returns an error (not supported).
+- If the key is structured, extracts the field and uses it to replace the entire key.
```

Comment:
I copied this from the docs in Conduit, but it doesn't appear it's true: https://github.com/ConduitIO/conduit/blob/71c6737846700e24a0cdb04c4fe1a890794b9276/pkg/processor/procbuiltin/extractfield.go#L72-L76.